bin/utils.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+const walkSync = require( "walk-sync" );
+const fs = require( "fs" );
+
+function getFilesFromArgs( args ) {
+  let globs = args.slice();
+
+  if ( !globs.length ) {
+    globs.push( "test/*.js" );
+  }
+
+  // For each of the potential globs, we check if it is a directory path and
+  // update it so that it matches the JS files in that directory.
+  globs = globs.map( glob => {
+    let stat;
+
+    try {
+      stat = fs.statSync( glob );
+    } catch ( e ) {}
+
+    if ( stat && stat.isDirectory() ) {
+      return `${glob}/*.js`;
+    } else {
+      return glob;
+    }
+  } );
+
+  const files = walkSync( process.cwd(), { globs } );
+
+  if ( !files.length ) {
+    error( "No files were found matching: " + args.join( ", " ) );
+  }
```

Comment:
I actually think this is useful feedback for users and ensures the process properly exits with a non-zero code in case they accidentally mess up an input path.